### DISCUSSION TITLE
Eldgos - 2023/6/6

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#32] [vefsida-ummæli]
# Eldgos - 2023/6/6

Trausti skellti sér á nýja eldgosið hjá Litla Hrút

https://www.biskupsreglan.com/verkefni/eldgos

**Comment by Ronaldo-is-better** [comment]
Sökkar

**Reply by TJ478** [reply]
Hvað ertu að tala um?
